deploy/scripts/src/btm/test.ts

```diff
@@ -102,21 +254,143 @@ const awsConfig = new AWSConfig({
+const payloadTimestampMin: number = Number(payloadTimestampArray[0])
+const payloadTimestampMax: number = Number(payloadTimestampArray[1])
+
+let messageBody: string
+let randomTimestamp: number
+let timestampFormatted: string
+
 const sqs = new SQSClient(awsConfig)
 
-export function sendEvent (): void {
-  const messageBody = eventData.payload.replace('UUID', uuidv4())
+export function sendEvent1 (): void {
+  randomTimestamp = Math.floor(Math.random() * (payloadTimestampMax - payloadTimestampMin + 1)) + payloadTimestampMin
+  timestampFormatted = new Date(randomTimestamp * 1000).toISOString()
+  messageBody = eventData.payload.replace('UUID', uuidv4())
+  messageBody = messageBody.replace('DATA1', payloadData1Array[0])
+  messageBody = messageBody.replace('DATA2', payloadData2Array[0])
+  messageBody = messageBody.replace('DATA3', payloadData3Array[0])
+  messageBody = messageBody.replace(/""timestamp"":\s*""\d+""/, `""timestamp"": ${randomTimestamp}`)
+  messageBody = messageBody.replace('TIMESTAMP', randomTimestamp.toString())
+  messageBody = messageBody.replace('TIMESTAMP_FORMATTED', timestampFormatted).replace('Z', '')
+  console.log('sendEvent1 === debug === messageBody \n', messageBody)
+  sqs.sendMessage(env.sqs_queue, messageBody)
+}
+
+export function sendEvent2 (): void {
+  randomTimestamp = Math.floor(Math.random() * (payloadTimestampMax - payloadTimestampMin + 1)) + payloadTimestampMin
+  timestampFormatted = new Date(randomTimestamp * 1000).toISOString()
+  messageBody = eventData.payload.replace('UUID', uuidv4())
+  messageBody = messageBody.replace('DATA1', payloadData1Array[1])
+  messageBody = messageBody.replace('DATA2', payloadData2Array[1])
+  messageBody = messageBody.replace('DATA3', payloadData3Array[1])
+  messageBody = messageBody.replace(/""timestamp"":\s*""\d+""/, `""timestamp"": ${randomTimestamp}`)
+  messageBody = messageBody.replace('TIMESTAMP', randomTimestamp.toString())
+  messageBody = messageBody.replace('TIMESTAMP_FORMATTED', timestampFormatted).replace('Z', '')
+  console.log('sendEvent2 === debug === messageBody \n', messageBody)
+  sqs.sendMessage(env.sqs_queue, messageBody)
+}
+
+export function sendEvent3 (): void {
+  randomTimestamp = Math.floor(Math.random() * (payloadTimestampMax - payloadTimestampMin + 1)) + payloadTimestampMin
+  timestampFormatted = new Date(randomTimestamp * 1000).toISOString()
+  messageBody = eventData.payload.replace('UUID', uuidv4())
+  messageBody = messageBody.replace('DATA1', payloadData1Array[2])
+  messageBody = messageBody.replace('DATA2', payloadData2Array[2])
+  messageBody = messageBody.replace('DATA3', payloadData3Array[2])
+  messageBody = messageBody.replace(/""timestamp"":\s*""\d+""/, `""timestamp"": ${randomTimestamp}`)
+  messageBody = messageBody.replace('TIMESTAMP', randomTimestamp.toString())
+  messageBody = messageBody.replace('TIMESTAMP_FORMATTED', timestampFormatted).replace('Z', '')
```

Comment:
```suggestion
  messageBody = messageBody.replace('TIMESTAMP_FORMATTED', timestampFormatted.replace('Z', ''))
```
The chaining of functions is in the wrong order here, otherwise it'll just replace the delete the first 'Z' character in the payload

Reply:
Done